### PR TITLE
Added Super User Functionality

### LINKED ISSUE #1747: Implement Super Admin User Feature
### Overview
With the introduction of admin user promotion and demotion capabilities in VRMS (see #1746), it’s critical that we implement safeguards to prevent the removal or modification of permissions by a rogue admin. To ensure the security of admin permissions, we’ve decided to create a dedicated super admin account ([vrms-admin-perm@hackforla.org](mailto:vrms-admin-perm@hackforla.org)) that is immune to modifications by other admin users. This account will have locked admin status, which cannot be altered. When another admin accesses the super admin’s EditUser page, the page will be view-only, preventing any changes while maintaining visibility.

### Action Items
- [x] Create user on Dev and Prod with ([vrms-admin-perm@hackforla.org](mailto:vrms-admin-perm@hackforla.org))
- [x] Promote user to Admin on Dev (Josh)
- [x] Promote user to Admin on Prod (Josh)
- [x] @ntrehan to discuss with @trillium & @jbubar how to implement this feature
- [x] Use email address to identify/filter user: vrms-admin-perm@hackforla.org (reasoning - this user has different names on Dev and Prod, but the same email is used - "VRMS ADMIN" on Prod, & "VRMS VRMS" on Dev currently).
- [ ] WHEN another admin user is viewing the super admin's "EditUser's" page, Disable features from being edited on the super admin's page (i.e., make the page "view-only" when viewed by another admin user):
  - [ ] "Is Active", 
  - [ ] "VRMS Admin", 
  - [ ] "Remove" Project function (once a project has been added to the user's profile, the user is , 
  - [ ] "Select a project" 
- [ ] WHEN the super admin user ([vrms-admin-perm@hackforla.org](mailto:vrms-admin-perm@hackforla.org)) is editing their own page, all of these features should be enabled.

### Resources/Instructions
- This issue is part of this epic: #1737

### PR DESCRIPTION
**Pull request by ntrehan**
Fixes #1747 

### What changes did you make and why did you make them ?

  - Disabled the ability to edit anything of the super user in the super user page
  - Added a new access level "superadmin" in the User Model
  - Some CSS changes to reflect that the buttons are disabled for a superadmin
  - Corrections from previous PR, "VRSM Admin" changed to "VRMS Admin" and the whole row is in one line
  - Please note that I promoted "mamoleo@gmail.com" to superadmin in vrms-test just for testing purposes

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)

<details>
<summary>Visuals before changes are applied</summary>

image

</details>

<details>
<summary>Visuals after changes are applied</summary>
  
image

</details>

### CODE REVIEW

File: client/src/components/user-admin/EditUsers.jsx [hunk 1]
```diff
@@ -9,6 +9,9 @@ const EditUsers = ({ userToEdit, backToSearch, updateUserDb, projects, updateUse
   const [isActive, setIsActive] = useState(userToEdit.isActive);
   const [isAdmin, setIsAdmin] = useState(userToEdit.accessLevel === "admin");
 
+  // Function to check if the current user is the super admin
```
Comment (trillium) [inline client/src/components/user-admin/EditUsers.jsx]: Semantics, but this comment says you provided a function but you've provided a boolean

**Review by trillium — APPROVED** [review]
This looks good, thanks for putting in the work to switch this around.

**Comment by trillium** [comment]
I approved the pull request, but can you please change the word function to boolean in the comment you provided

File: client/src/components/user-admin/EditUsers.jsx (hunk 1 above)
Comment (ntrehan) [reply client/src/components/user-admin/EditUsers.jsx]: Good catch trillium! I changed approach mid way but forgot to change the comment. Addressing it right away

**Comment by trillium** [comment]
Merged!

**Comment by trillium** [comment]
Merged!

**Pull request merged**